nss-cohort-36/bangazon-client-virtual-magicians-1
# Jg product 1

# Description
Added the home page to where it contains a location for the welcome and the as well as a location for product locations filter.

Closes #[Enter issue #]
## Type of change

Please delete options that are not relevant.
- [x ] New feature (non-breaking change which adds functionality)

# Testing Instructions
Make sure to commit local changes in your branch before switching to master.
1. `git checkout master`
2. `git fetch --all`
3. `git checkout jg-product-1`
4. go to our home page of our application and ensure that you can see the nav bar as well as the home page below

# Checklist:
